packages/components/psammead-media-player/src/Guidance/index.jsx

```diff
@@ -37,15 +49,35 @@ const GuidanceMessage = styled.strong`
   }
 `;
 
-const Guidance = ({ guidanceMessage, service }) => (
-  <GuidanceWrapper service={service}>
-    <GuidanceMessage aria-hidden="true">{guidanceMessage}</GuidanceMessage>
+const StyledNoScript = styled.noscript`
+  position: absolute;
+  bottom: 0;
+`;
+
+const Guidance = ({ guidanceMessage, service, type }) => (
+  <GuidanceWrapper service={service} guidanceMessage={guidanceMessage}>
+    {guidanceMessage && (
+      <GuidanceMessage className="guidance-message" aria-hidden="true">
+        {guidanceMessage}
+      </GuidanceMessage>
+    )}
+    <StyledNoScript>
+      <GuidanceMessage>
+        This {type} cannot play in your browser. Please enable Javascript or
```

Comment:
@sareh do you think we could opt this as default and still pass it as prop for now till such a time when the translations are available ?

Reply:
To keep them consistent with other components, it would be best to make this a required prop, adding this example to the readme.
Another benefit is that we can add the translation field into Simorgh now, and request translations from product / editorial teams.

Reply:
done.